### PR TITLE
Updated for CDK and Terraform Documentation

### PR DESCRIPTION
**Pull request by troy-ameigh**
*Description of changes:*

I have added to the CDK support, and then added the same setup for Terraform docs. I have also configured so you can create a 2nd Lang with either. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by MarciaRieferJohnston — DISMISSED** [review]

**Review by dowen12 — APPROVED** [review]

File: .utils/create_repo_structure.sh [hunk 1]
```diff
@@ -2,21 +2,31 @@
 # # Work in progress.
 # exit 1
 
-#Adds Help and Second Language options (-h | -l)
-while getopts hl  option
+#Adds Otions for the following: ((Help/Second Language/CDK/Terraform options) (-h | -l | -c | -t))
```
Comment (MarciaRieferJohnston) [inline .utils/create_repo_structure.sh:5]: "Options" :)

**Review by MarciaRieferJohnston — COMMENTED** [review]
I noted a typo ("Otions") in a comment on `create_repo_structure.sh`. Later learned—at the wonderful Write the Docs conference—that I could suggest the edit in my own commit. So I did that on April 30. Cool.

File: .utils/create_repo_structure.sh (hunk 1 above)
Comment (MarciaRieferJohnston) [inline .utils/create_repo_structure.sh:5]:
```suggestion
#Adds Options for the following: ((Help/Second Language/CDK/Terraform options) (-h | -l | -c | -t))
```

**Review by MarciaRieferJohnston — APPROVED** [review]

File: .specific/.cdk/cost.adoc [hunk 2]
```diff
@@ -0,0 +1,10 @@
+
+You are responsible for the cost of the AWS services and any third-party licenses used while running this Quick Start. There is no additional cost for
+using the Quick Start.
+
+The AWS CDK deployment for Quick Starts could include
+configuration parameters that you can customize. Some of the settings,
+such as the instance type, affect the cost of deployment. For cost estimates, 
+see the pricing pages for each AWS service you use. Prices are subject to change. 
+
+TIP: After you deploy the Quick Start, https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/billing-reports-gettingstarted-turnonreports.html[create AWS Cost and Usage Reports^] to deliver billing metrics to an Amazon Simple Storage Service (Amazon S3) bucket in your account. These reports provide cost estimates based on usage throughout each month and aggregate the data at the end of the month. For more information, see https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/billing-reports-costusage.html[What are AWS Cost and Usage Reports?^]
```
Comment (MarciaRieferJohnston) [inline .specific/.cdk/cost.adoc:10]:
```suggestion
TIP: After you deploy the Quick Start, create  https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/billing-reports-gettingstarted-turnonreports.html[AWS Cost and Usage Reports^] to deliver billing metrics to an Amazon Simple Storage Service (Amazon S3) bucket in your account. These reports provide cost estimates based on usage throughout each month and aggregate the data at the end of the month. For more information, see https://docs.aws.amazon.com/cur/latest/userguide/what-is-cur.html[What are AWS Cost and Usage Reports?^]
```

File: .specific/.terraform/cost.adoc [hunk 3]
```diff
@@ -0,0 +1,10 @@
+
+You are responsible for the cost of the AWS services and any third-party licenses used while running this Quick Start. There is no additional cost for
+using the Quick Start.
+
+The Terraform deployment for Quick Starts could include
+configuration parameters that you can customize. Some of the settings,
+such as the instance type, affect the cost of deployment. For cost estimates, 
+see the pricing pages for each AWS service you use. Prices are subject to change. 
+
+TIP: After you deploy the Quick Start, https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/billing-reports-gettingstarted-turnonreports.html[create AWS Cost and Usage Reports^] to deliver billing metrics to an Amazon Simple Storage Service (Amazon S3) bucket in your account. These reports provide cost estimates based on usage throughout each month and aggregate the data at the end of the month. For more information, see https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/billing-reports-costusage.html[What are AWS Cost and Usage Reports?^]
```
Comment (MarciaRieferJohnston) [inline .specific/.terraform/cost.adoc:10]:
```suggestion
TIP: After you deploy the Quick Start, create https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/billing-reports-gettingstarted-turnonreports.html[AWS Cost and Usage Reports^] to deliver billing metrics to an Amazon Simple Storage Service (Amazon S3) bucket in your account. These reports provide cost estimates based on usage throughout each month and aggregate the data at the end of the month. For more information, see https://docs.aws.amazon.com/cur/latest/userguide/what-is-cur.html[What are AWS Cost and Usage Reports?^]
```

File: .specific/.cdk/deploy_steps.adoc [hunk 4]
```diff
@@ -0,0 +1,39 @@
+// We need to work around Step numbers here if we are going to potentially exclude the AMI subscription
+=== Planning the deployment
+
+This Quick Start requires a moderate to high-level understanding of how to manage {partner-product-short-name}. This deployment also requires a moderate understanding of the following: https://docs.aws.amazon.com/servicequotas/latest/userguide/intro.html[Service Quotas^], https://docs.aws.amazon.com/cdk/latest/guide/home.html[AWS Cloud Development Kit (AWS CDK)^], and https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/Welcome.html[AWS CloudFormation^].
+
+The deployment process takes about {deployment_time} to complete, and the template launches in the {default_deployment_region} Region by default.
+
+=== Launch the Quick Start
+To deploy this Quick Start, use AWS CDK. For more information, see https://docs.aws.amazon.com/cdk/latest/guide/work-with.html[Working with the AWS CDK].
```
Comment (MarciaRieferJohnston) [inline .specific/.cdk/deploy_steps.adoc:9]:
```suggestion
To deploy this Quick Start, use AWS CDK. For more information, see https://docs.aws.amazon.com/cdk/latest/guide/work-with.html[Working with the AWS CDK^].
```

Comment (MarciaRieferJohnston) [reply .specific/.cdk/deploy_steps.adoc:9]: The "^" opens the link in a new browser tab.

**Review by MarciaRieferJohnston — CHANGES_REQUESTED** [review]
I made small edits in several places (a typo fix, URL updates, etc.)

**Review by MarciaRieferJohnston — APPROVED** [review]

**Review by MarciaRieferJohnston — APPROVED** [review]

File: .utils/create_repo_structure.sh [hunk 5]
```diff
@@ -2,21 +2,31 @@
...
 
-#Adds Help and Second Language options (-h | -l)
-while getopts hl  option
+#Adds Options for the following: ((Help/Second Language/CDK/Terraform options) (-h | -l | -c | -t))
+while getopts hlct  option
 do
     case "${option}" in
       h )
           echo "Usage:"
           echo "Run './create_repo_structure.sh' with no options for English langauge only."
```
Comment (MarciaRieferJohnston) [inline .utils/create_repo_structure.sh:11]:
```suggestion
          echo "Run './create_repo_structure.sh' with no options for English language only."
```

File: .utils/create_repo_structure.sh [hunk 6]
```diff
@@ -2,21 +2,31 @@
...
           echo "Run './create_repo_structure.sh -l' to add files for second langauge."
```
Comment (MarciaRieferJohnston) [inline .utils/create_repo_structure.sh:12]:
```suggestion
          echo "Run './create_repo_structure.sh -l' to add files for second language."
```

**Review by MarciaRieferJohnston — APPROVED** [review]

**Review by MarciaRieferJohnston — APPROVED** [review]

**Pull request merged**